packages/Webkul/Admin/src/Resources/assets/js/components/navigation/notification-list.vue

```diff
@@ -0,0 +1,210 @@
+                            <div class="notif-icon processing">
+                                <span :class="ordertype.processing.icon"></span>
+                            </div>                   
+                            <div class="notif-content">
+                                <a :href="`${orderViewUrl}${notification.order_id}`">
+                                    #{{ notification.order.id }} {{ordertype.processing.message}}
+                                </a>
+                            </div>
+                            <div class="notif-content">
+                                {{ moment(notification.order.created_at).fromNow() }}
+                            </div>
+                        </template>
+                        <template v-if="notification.order.status == 'closed'">
+                            <div class="notif-icon closed">
+                                <span :class="ordertype.closed.icon"></span>
+                            </div>                
+                            <div class="notif-content">
+                                <a :href="`${orderViewUrl}${notification.order_id}`">
+                                    #{{ notification.order.id }} {{ordertype.closed.message}}
+                                </a>
+                            </div>
+                            <div class="notif-content">
+                                {{ moment(notification.order.created_at).fromNow() }}
+                            </div>
+                        </template>
+                    </li>
+                </ul>
+                <pagination align="center" :data="pagNotif" @pagination-change-page="getResults">
+                    <span slot="prev-nav">&lt;</span>
+	                <span slot="next-nav">&gt;</span>
+                </pagination>
+                <ul class="notif" v-if="notifications.length == 0">
+                    <li>{{ noRecordText }}</li>
+                </ul>
+                
+            </div>
+        </div> 
+</template>
+
+<script>
+    
+    export default {  
+
+        props: [
+            'url',
+            'orderViewUrl',
+            'pusherKey',
+            'pusherCluster',
+            'title',
+            'orderStatus',
+            'noRecordText'   
+        ],
+
+        data: function() {
+            return {
+                notifications: [],
+                pagNotif: {},
+                id: '',
+                status: '',
+                ordertype: {
+                    pending : {
+                        icon: 'pending-icon',
+                        message: 'Order Pending'
+                    },
+                    processing : {
+                        icon: 'processing-icon',
+                        message: 'Order Processing'
+                    },
+                    canceled : {
+                        icon: 'canceled-icon',
+                        message: 'Order Canceled'
```

Comment:
Please use translations instead of static text

Reply:
ok sir